src/Data/Profunctor/Linear.hs

```diff
@@ -58,7 +59,10 @@ class (SymmetricMonoidal m u, Profunctor arr) => Strong m u arr where
   {-# INLINE second #-}
 
 class (Strong (,) () arr, Strong Either Void arr) => Wandering arr where
-  wander :: Data.Traversable f => a `arr` b -> f a `arr` f b
+  -- | Equivalently but less efficient in general:
+  --
+  -- > wander :: Data.Traversable f => a `arr` b -> f a `arr` f b
+  wander :: forall s t a b. (forall f. Control.Applicative f => (a #-> f b) -> s #-> f t) -> a `arr` b -> s `arr` t
```

Comment:
I like this much better actually.

Reply:
In principle, I like the previous one better. Like, it's theoretically less ad hoc. But on the other hand, it's also less general in a sense.